randovania/games/fusion/gui/dialog/game_export_dialog.py

```diff
@@ -25,19 +29,69 @@ class FusionGameExportDialog(GameExportDialog, Ui_FusionGameExportDialog):
     def game_enum(cls) -> RandovaniaGame:
         return RandovaniaGame.FUSION
 
+    def __init__(self, options: Options, patch_data: dict, word_hash: str, spoiler: bool, games: list[RandovaniaGame]):
+        super().__init__(options, patch_data, word_hash, spoiler, games)
+
+        self._base_output_name = f"MARS - {word_hash}"
+        per_game = options.options_for_game(self.game_enum())
+        assert isinstance(per_game, FusionPerGameOptions)
+
+        # Input
+        self.input_file_button.clicked.connect(self._on_input_file_button)
+
+        # Output
+        self.output_file_button.clicked.connect(self._on_output_file_button)
+
+        if per_game.input_path is not None:
+            self.input_file_edit.setText(str(per_game.input_path))
+
+        if per_game.output_path is not None:
+            output_path = per_game.output_path.joinpath(self.default_output_name)
+            self.output_file_edit.setText(str(output_path))
+
+        add_field_validation(
+            accept_button=self.accept_button,
+            fields={
+                self.input_file_edit: lambda: False,
+                self.output_file_edit: lambda: False,
+            },
+        )
+
+    @property
+    def valid_input_file_types(self) -> list[str]:
+        return ["gba"]
+
+    @property
+    def valid_output_file_types(self) -> list[str]:
+        return ["gba"]
```

Comment:
do we ever plan to support anything *other* than gba types? if not, we could probably change these to just return strings instead of list of strings.

Reply:
outside of my understanding of gba formats. does it hurt anything the way it is beyond could be more appropriately typed?

Reply:
on second thought, no, because `prompt_for_output_file` needs a list anyway.